coreos-iso-embed-ignition

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python3
+
+import argparse
+import gzip
+import os
+import shutil
+import struct
+import subprocess
+import sys
+import tempfile
+
+FILENAME = 'config.ign'
+
+class EmbedException(Exception):
+    pass
+
+
+class CopiedFileHolder:
```

Comment:
what value is this class giving us?

Reply:
basically garbage collection if we error out in the middle?

Reply:
Yup.